kernel/memory/Pmm.cpp

```diff
@@ -229,6 +235,34 @@ namespace Npk::Memory
             InsertRegion(zones[1], base, length);
     }
 
+    PageInfo* PMM::Lookup(uintptr_t pfn)
+    {
+        PmZone* zone = nullptr;
```

Comment:
not sure if you are doing it or not, but i tend to leave a blank space between the variables declaration and the rest of the code.